index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul class="first-navbar">
+                <li><a class="first-navbar1" href="#pages"> Pages</a></li>
+                <li>Logout</li>
+            </ul>
+            <div class="ham-cross">
+                <span class="hamburger-icon1">&#9776</span>
+                    <span class="cross hidden">X</span>
+            </div>
+            <div class="logo-items">
+                <span class="nav-logo"> <a href="index.html"> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                    <ul class="second-navbar">
+                        <li class="second-navbars-links"><a href="about.html">About</a></li>
+                        <li class="second-navbars-links"><a href="#"> Products</a></li>
+                        <li class="second-navbars-links"><a href="#"> News</a></li>
+                        <li class="second-navbars-links"><a href="#"> login</a></li>
+                    </ul>
+                </div>
+        <div class="second-navbar-popup hidden">
+            <ul class="second-navbar-popup-items">
+                <li class="second-navbar-popup-links"><a href="about.html">About</a> </li>
```

Comment:
- Your previous reviewer requested for some design modification for this dark top bar & navbar but after going through your code base it still doesnt match the provided project design as shown below. 
- [ ] For the top bar, it is to be placed at the right side of the viewport and it should contain `social media icons` as well as the links as shown in the image below
- [ ] For the navbar the `login` have a red border just like the `campaign link` from the project design.
- please ensure that before resubmitting this requirements are met so that your project can be approved, Good job implementing all of the previous reviews as well :+1:  
- project design
![top-b](https://github.com/alizaamir123/Capstone-project/assets/89055502/9fc50e51-cc86-4141-9be3-fc6a4874e5d0)
- your design
![top-s](https://github.com/alizaamir123/Capstone-project/assets/89055502/3e278f3f-abd5-481d-8b87-4602518ed413)